### DISCUSSION TITLE
Prefab이 깨지는 문제

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/XRStudy#42] [7. Q&A]
![Screenshot 2024-06-27 at 11 31 56 PM](https://github.com/hemil0102/XRStudy/assets/83139316/b02b950e-a7a1-40ef-93ef-a4fc2fe12746)

그림과 같이 Unity에서 제공하는 파일들을 Import하였는데 분홍색으로 깨집니다. 
외부 에셋을 다운 받다보면 꽤 많이 발생하는 문제 같은데, 어떻게 절차적으로 확인해서 수정해볼 수 있을까요?

**Comment by hemil0102** [comment]
이 부분은 Unity에서 제공하는 Prefab들이 URP 프로젝트에서는 제대로 불러와집니다. 
따라서 프로젝트를 다시 생성하는 것이 좋아보이는데, 다른 방법이 있는지 찾아보는 중입니다.

**Comment by hemil0102** [comment]
Material Shader를 Standard로 변경하니 해결되었습니다. 
다만 하나 하나 Shader를 변경하는 방법외에 전체 Shader를 한 번에 다 바꿔주는게 있는지 궁금합니다.

**Comment by jongfeel** [comment]
URP 프로젝트는 Universal Render Pipeline / Lit 으로 하는게 기본 설정일 겁니다.

처음부터 URP 프로젝트로 생성하고 하면 이런 문제는 없긴 해요.
